src/CommunityVoices/Model/Mapper/LocationCollection.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace CommunityVoices\Model\Mapper;
+
+use PDO;
+use CommunityVoices\Model\Component\DataMapper;
+use CommunityVoices\Model\Entity;
+
+class LocationCollection extends DataMapper
+{
+    public function fetch(Entity\LocationCollection $container)
+    {
+        $this->fetchAll($container);
+    }
+
+    public function fetchAll(Entity\LocationCollection $container)
```

Comment:
change naming -- these are collections (composites) `$locationCollection` is better.

Reply:
I figure https://github.com/EnvironmentalDashboard/community-voices/pull/19/commits/532e5b7e29d28ad98af83787c97ec8ffbeddfa8f covers what you mean?